.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@steiner2001 Make sure to remove the .vs folder, as once it's checked in it's not enough to just .gitignore the files. (Look into the `git rm` command)